www/video/compare.php

```diff
@@ -446,14 +446,27 @@
                             } else {
                                 echo '<h2>Experiment Results</h2>';
                                 echo '<div class="experiment_meta">';
-                                echo '<div class="experiment_meta_included">';
+                                echo '<div class="experiment_meta_included" id="experimentsAppliedSection">';
                                 echo '<p>Experiments Applied:</p>';
                                 echo '<ul>';
                                 include __DIR__ . '/../experiments/list_applied.inc';
 
                                 echo "<li><a class=\"experiment_meta-more\" href=\"" . $experimentOriginalExperimentsHref . "\">Experiment More</a></li>";
                                 echo '</ul>';
                                 echo '</div>';
+
+                                echo "<script>
```

Comment:
@v-hclaveria  is there a reason for which this is here ? we already have multiple script tag, can this be added there?

Reply:
Moved as talked on stand-up